esm-samples/rollup/rollup.config.js

```diff
@@ -15,7 +17,15 @@ export default {
     format: "es"
   },
   plugins: [
-    del({ targets: "public/chunks", runOnce: true, verbose: true }),
+    clean(),
+    del({ targets: ["public/chunks", "public/assets"], runOnce: true, verbose: true }),
```

Comment:
Do we need `clean` and `del`?

Reply:
`del` is a nice failsafe. It insures assets version matches the API.
`clean` would be more of a best practice and mirrors the webpack plugin sample.

Reply:
- `clean` doesn't appear to be maintained: https://github.com/alwaysonlinetxm/rollup-plugin-clean
- I'm not clear what it's doing that `del` isn't doing?

Reply:
Oh, good catch. Didn't realize that. I'll remove `clean`